oras/auth/base.py

```diff
@@ -81,8 +93,9 @@ def _load_auth(self, hostname: str) -> bool:
             return True
         return False
 
-    @decorator.ensure_container
-    def load_configs(self, container: container_type, configs: Optional[list] = None):
+    def load_configs(
+        self, container: oras.container.Container, configs: Optional[list] = None
```

Comment:
I changed the container type as it relied on `self.get_container()` method above trough the decorator. Which as said would not have worked anyways